### PR TITLE
Add Close button for login/register modals in Demo

### PR DESCRIPTION
**Pull request by melux**
Login and register modals in demo mode miss close button, let's fix that.
Will fixe https://github.com/egovernment/eregistrations-salvador/issues/1718

### CODE REVIEW

**Comment by melux** [comment]
@medikoo ready for review, merge and update. Especially awaited on ELS and GT. Thanks!

File: view/components/login-dialog.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ var db        = require('mano').db
 module.exports = dialog(
 	{ id: 'login', class: 'dialog-login dialog-modal' },
 	header(
+		a({ class: 'close', type: 'button', href: "." }, i({ class: 'fa fa-close' })),
```
Comment (medikoo) [inline view/components/login-dialog.js]: Introduced class,  doesn't follow name conventions as described here: http://help.eregistrations.org/framework/styles/#class-naming-conventions

Comment (melux) [inline view/components/login-dialog.js]: yes renamed!

**Comment by medikoo** [comment]
Merged into all systems

**Pull request merged**